dd-sdk-android/src/main/kotlin/com/datadog/android/rum/internal/monitor/DatadogRumMonitor.kt

```diff
@@ -126,12 +123,35 @@ internal class DatadogRumMonitor(
         GlobalRum.updateViewId(null)
     }
 
+    @Synchronized
+    override fun addUserAction(
```

Comment:
what is we have unclosedresources in this moment ? Don't we want to send those first with the previous action event ?

Reply:
What if I call add user action 2 times in the same view and without any resource loading in between ? Shouldn't we send the last one first before adding the new one ?

Reply:
What will happen if you I add an user action in a view and will never be sent (no resource loading no nothing) and we are never switching to another view ...I think this User action will not be sent in that moment. Maybe we should call "sendUnclosedAction" also on the "stopView" or maybe you should have a handler which periodically checks for unsent expired actions and send them. I would also add unit tests for this case and the one above.

Reply:
The rule on browser (for now) is if an action is still "active", we ignore the new one.
I'll handle this case.